src/EditorFeatures/CSharpTest/Completion/CompletionProviders/Snippets/AbstractCSharpConditionalBlockSnippetCompletionProviderTests.cs

```diff
@@ -467,5 +467,36 @@ void M(bool arg)
 
             await VerifyItemIsAbsentAsync(markupBeforeCommit, ItemToCommit);
         }
+
+        [WpfFact]
+        public async Task CorrectlyDealWithLeadingTriviaInInlineSnippetTest()
+        {
+            var markupBeforeCommit = """
+                class Program
+                {
+                    void M(bool arg)
+                    {
+                        // comment
```

Comment:
💡 Consider adding tests for other types of trivia (`#region`, `#if true`, `#pragma warning disable`)

Reply:
Note, that due to the fact that inserted snippet node is formatted, different trivia behave a bit differently (for instance, `#if` directive is put at the start of a line), but I think such behaviour is expected

Reply:
Yes that would be expected